app/views/admin/shelters/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Admin Shelters Index</h1>
+
+<% @shelters.each do |shelter|%>
+  <%= shelter.name%>
+<% end %>
+
+<h3>Shelters with Pending Applications</>
+<div id="pending_applications">
+  <% Shelter.pending_applications.each do |shelter|%>
```

Comment:
We call model instance methods from views all the time, but when it comes to class methods, it's typically more conventional to call the class method from the controller. In the controller action, you could do: `@shelters_with_pending_apps = Shelter.pending_applications` and then iterate on that instance variable in the view.